test/integration-test/sql-parser/src/test/java/org/apache/shardingsphere/sql/parser/base/DynamicLoadingSQLParserParameterizedTest.java

```diff
@@ -133,7 +134,8 @@ public final void assertParseSQL() {
         try {
             ParseASTNode parseASTNode = new SQLParserEngine(databaseType, new CacheOption(128, 1024L)).parse(sql, false);
             new SQLVisitorEngine(databaseType, "STATEMENT", true, new Properties()).visit(parseASTNode);
-        } catch (final SQLParsingException | ClassCastException | NullPointerException | SQLASTVisitorException | NumberFormatException | StringIndexOutOfBoundsException ignore) {
+        } catch (final SQLParsingException | ClassCastException | NullPointerException | SQLASTVisitorException
+                | NumberFormatException | StringIndexOutOfBoundsException | ServiceProviderNotFoundServerException ignore) {
```

Comment:
there are too much exception caught, looks a little bit chaos.
if this will be ignored, is it possible to catch Exception ?

Reply:
make sense. )

Reply:
Exception and RuntimeException are not allowed.
To make muli-exceptions easier to read, I replace SQLParsingException & SQLASTVisitorException with KernelSQLException.